src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -672,4 +760,155 @@ mod tests {
             assert_eq!(output, expected, "lhs={:?}, rhs={:?}", lhs, rhs);
         }
     }
+
+    #[test]
+    fn test_int_divide_int() {
+        let test_cases = vec![
+            (13, false, 11, false, Some(1), false),
```

Comment:
is_err is always false, no need?

Reply:
> is_err is always false, no need?

Oh, there was some error, but now there is none. I'll fix it.